werkzeug/http.py

```diff
@@ -340,8 +340,10 @@ def parse_options_header(value, multiple=False):
     :return: (mimetype, options) or (mimetype, options, mimetype, options, …)
              if multiple=True
     """
+    if value is None or not value.strip(' ,;'):
+        return '', {}
 
-    if not value:
+    if value.startswith(',') or value.startswith(';'):
```

Comment:
I've investigated a bit further. You can get an empty result only in the cases where the ``_option_header_start_mime_type`` regular expression doesn't give a match. This boils down to any value not matching ``\s*([^;,\s]+)([;,]\s*.+)?`` which include any whitespace stripped value starting with a comma or semi-colon (e.g. `` , a ``).

I'm not sure if it's worthwhile having this second if-clause here - for correct content types it will add extra computation time, and only reduce computation time for invalid content types.

Let me know what's your preference, and I'll update the PR accordingly.